### DISCUSSION TITLE
StructureSelectionQuery for aa + surroundings

### DISCUSSION THREAD

**Discussion by dhrblav** [molstar/molstar#372] [Q&A] [answered]
Hi,
I am trying to create a StructureSelectionQuery including a given amino acid of a protein + all its surroundings, including ligands (and possibly components of other macromolecules).
Can you please advise on how to do this?
Thank you,
Andre

**Comment by dsehnal** [comment, marked as answer]
Hi, here's an example of creating such selections https://github.com/molstar/molstar/blob/master/src/examples/proteopedia-wrapper/index.ts#L358

**Reply by dhrblav** [reply]
Great, thank you.
